Library/Homebrew/utils/puts_columns.rb

```diff
@@ -0,0 +1,41 @@
+require "utils/tty"
+
+module Kernel
```

Comment:
It needs to be in `Kernel` to work with `IO` objects, i.e. `$stderr.puts_columns`.